src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static void getRandomColor() {
+        Random random = new Random();
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
You should use the 'index' to fetch and return a color from the Color enum. The current method does not use the 'index' variable.